agent/xds/listeners.go

```diff
@@ -905,6 +909,14 @@ func makeListenerFromUserConfig(configJSON string) (*envoy_listener_v3.Listener,
 	return &l, nil
 }
 
+func injectConnectionBalanceConfig(balanceType string, listener *envoy_listener_v3.Listener) {
+	if balanceType == "exact_balance" {
```

Comment:
It's also probably worth logging here if `balanceType` is anything but `""` or `"exact_balance"`.